### ISSUE TITLE
Console spam in Windows

### ISSUE THREAD

**Issue by iarspider** [DoubleDoorDevelopment/D3Backend#12]
On Windows, this line is constantly added to the log:
```
[WARN - cmd] java.io.IOException: The handle is invalid
```

**Comment by CheshireKatt** [comment]
I'm getting the same issue with D3Backend-0.9.0.49.jar, on Windows 7 x64.  I get it with a fresh setup -- download D3Backend-0-9.0.49.jar and launch it from explorer in an empty folder.  It creates the logs and servers folders.  From there, I browsed to localhost, created my account, entered the admin token, created a server via the tool, then copied my server files in there, and started it.  Still get it.  Tried stopping the backend (by killing it in task manager?  Is there a "right" way to do that?), no change, as soon as I start it back up they begin again.

I'm entirely open to the possibility I'm using it wrong, but this generates me megs of logfile VERY quickly so if you've got any ideas I'm all ears.  :)

As an example of how frequently I get the messages, here's a bit of my log:

2015-04-11 12:56:13,343 [WARN - cmd] java.io.IOException: The handle is invalid
2015-04-11 12:56:13,344 [WARN - cmd] java.io.IOException: The handle is invalid
2015-04-11 12:56:13,345 [WARN - cmd] java.io.IOException: The handle is invalid
2015-04-11 12:56:13,345 [WARN - cmd] java.io.IOException: The handle is invalid
2015-04-11 12:56:13,346 [WARN - cmd] java.io.IOException: The handle is invalid
2015-04-11 12:56:13,347 [WARN - cmd] java.io.IOException: The handle is invalid
2015-04-11 12:56:13,347 [WARN - cmd] java.io.IOException: The handle is invalid
2015-04-11 12:56:13,348 [WARN - cmd] java.io.IOException: The handle is invalid
2015-04-11 12:56:13,349 [WARN - cmd] java.io.IOException: The handle is invalid

**Comment by dries007** [comment]
You need to open the backend with command line, then you can stop it with the command interface.
Perhaps that is causing the issue...
Can you run it through cmd and report back?

**Comment by Claycorp** [comment]
@dries007 Should this be changed to allow for this type of launching or would that just be too complicated for whats its worth?

**Comment by dries007** [comment]
I might make a window thing like MC has, idk yet. Not high on priority list

**Comment by Claycorp** [comment]
Im glad to see people using this too!

**Comment by CheshireKatt** [comment]
Aha.  That did it, a simple java -jar D3Backend-0.9.0.49.jar has resolved the console spam issue.  The console appears to be properly responsive with a default MineCraft install too, so I appear to be good to go.

What's the proper way from within the web interface to shut down the d3backend instance itself?  I see how to stop individual servers, but not the entire backend.

**Comment by dries007** [comment]
Ok, that explains that. 
You cannot shut the backend down from the web interface, only from the console. (use "help" in the console for more information)

**Comment by CheshireKatt** [comment]
I see now -- I'd assumed the console window was output-only and that I only had access to the server consoles from the pop-up server console from the web interface.  Help in the console I launched the backend from in the first place does exactly what I'd want (and then some).  Thanks again!  I've been really looking forward to giving this a try for my next server, and this looks like all my startup issues are now resolved.
